software/database/src/main/java/org/apache/brooklyn/entity/database/postgresql/PostgreSqlSshDriver.java

```diff
@@ -294,7 +294,9 @@ public void customize() {
 
         // Wait for commands to complete before running the creation script
         DynamicTasks.waitForLast();
-
+        if(entity.getConfig(PostgreSqlNode.INITIALIZE_DB)){
```

Comment:
This will throw a null pointer exception if `INITIALIZE_DB` is null. You can avoid this with the slightly ungainly `if (Boolean.TRUE.equals(getConfig(..))`.